schemas/manifest.cue

```diff
@@ -58,7 +58,7 @@
         "pinned": {
             "1804": "1.7.1-1" // default in 1804 vhds.
         }
-        "edge": "1.7.7-1",  // edge is default in vhd.
+        "edge": "1.7.14-1",  // edge is default in vhd.
```

Comment:
for future reference, we only ever need to update this manifest.cue - `make generate` as well as the VHD build itself will use this file to `export` to manifest.json in cloud-init/artifacts

Reply:
yep, anuj informed me a while back, sorry should've reported back